src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -106,15 +106,23 @@ private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmis
             if (UtilsValidators.isNotNull(trustCeasedDate, qualifiedFieldName, errors, loggingContext)) {
                 DateValidators.isDateInPast(trustCeasedDate, qualifiedFieldName, errors, loggingContext);
                 DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeasedDate, creationDate, qualifiedFieldName, errors, loggingContext);
-            }
+
+                if (trustDataDto.getIndividuals() != null && !trustDataDto.getIndividuals().isEmpty()) {
+                    DateValidators.isCeasedDateOnOrAfterIndividualsDateOfBirth(trustCeasedDate, trustDataDto.getIndividuals(), qualifiedFieldName, errors, loggingContext);
+                }
+           }
         }
     }
 
-    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, String trustId) {
+    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, String trustId, boolean istrustInvolvedInOverseasEntity) {
         // Use the trust id to whizz through all the Beneficial Owners that are associated with this trust. As soon as a
         // matching BO is found where cease date is null and a trust NOC is set then this indicates that the trust is
         // still required
 
+        if (!istrustInvolvedInOverseasEntity) {
```

Comment:
This new `if` check could be combined with the one below (using an OR operand), as outcome the same.

Reply:
Combined the checks